src/main/sensors/barometer.c

```diff
@@ -161,7 +163,22 @@ void baroPreInit(void)
 #endif
 }
 
-bool baroDetect(baroDev_t *baroDev, baroSensor_e baroHardwareToUse)
+static bool baroDetect(baroDev_t *baroDev, baroSensor_e baroHardwareToUse);
+
+void baroInit(void)
+{
+    baroDetect(&baro.dev, barometerConfig()->baro_hardware);
+
+    const float cutoffHz = barometerConfig()->baro_noise_lpf / 100.0f;
+    // const float cutoffHz = TASK_BARO_RATE_HZ / 4; // -3dB cutoff at nyquistFreq / 2
```

Comment:
Remove comment or move to end of line where it belongs.

Reply:
You can remove it completely. This was meant to be the highest cutoff you can use while guaranteeing not to get aliasing effects from the stepped inputs.